src/patmat.ml

```diff
@@ -31,6 +31,13 @@ let mex a =
     n
   with Brk i -> i
 
+let filteri p l =
```

Comment:
Can you add a comment above so we don't forget to remove this when we upgrade to 4.11?